.gitignore

```diff
@@ -1,3 +1,7 @@
 .pyc
 .ipynb_checkpoints/
 *.nbconvert.ipynb
+photutils/dev/*
+photutils/02_photutils_source_detection.ipynb
```

Comment:
(copying from Slack conversation) 

What I’ve been doing so far is I have one branch, just called `photutils`, where I have been developing all of the notebooks. I just created this PR branch, `photutils-background-estimation`, off of that development branch to ultimately merge the first notebook into `master`.

You suggest having one branch per notebook, but I’ve found that pretty frequently I need to make changes to multiple or all of the notebooks, not just one at a time. Do you think it sounds like a feasible workflow for me to keep this main development branch, then I can just branch off and delete the irrelevant notebooks whenever I submit a PR?

Reply:
Probably not many people will be looking at this other than us, but just in case... we concluded in an out-of-band discussion that we should change to a separate branch per notebook and make separate commits in the individual branches.